### PR TITLE
Add SDK-SP1-Updates-test repo 

### PR DESCRIPTION
**Pull request by vmoravec**
...to avoid conflicts if test updates channel is enabled.
Follow-up for https://github.com/SUSE-Cloud/automation/pull/1849

### CODE REVIEW

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -440,6 +440,12 @@ function add_sdk_repo
         6)
             zypper ar -p $sdk_repo_priority -f $smturl/SUSE/Products/SLE-SDK/12-SP1/$arch/product/ SDK-SP1
             zypper ar -p $sdk_repo_priority -f $smturl/SUSE/Updates/SLE-SDK/12-SP1/$arch/update/ SDK-SP1-Update
+
+            if [ -n "$want_test_updates" -a "$want_test_updates" != "0" ] && isrepoworking SLE12-SP1-SDK-Updates-test ; then
+                 add_mount "SLE12-SP1-SDK-Updates-test" \
```
Comment (bmwiedemann) [inline scripts/qa_crowbarsetup.sh]: needs to be indented with 4 spaces, not 5

**Comment by vmoravec** [comment]
@bmwiedemann wrong indentation fixed

File: scripts/qa_crowbarsetup.sh [hunk 2]
```diff
@@ -440,6 +440,12 @@ function add_sdk_repo
         6)
             zypper ar -p $sdk_repo_priority -f $smturl/SUSE/Products/SLE-SDK/12-SP1/$arch/product/ SDK-SP1
             zypper ar -p $sdk_repo_priority -f $smturl/SUSE/Updates/SLE-SDK/12-SP1/$arch/update/ SDK-SP1-Update
+
+            if [ -n "$want_test_updates" -a "$want_test_updates" != "0" ] && isrepoworking SLE12-SP1-SDK-Updates-test ; then
```
Comment (bmwiedemann) [inline scripts/qa_crowbarsetup.sh]: IMHO, an easier check would be [[ $want_test_updates = 1 ]]

Comment (vmoravec) [inline scripts/qa_crowbarsetup.sh]: It's a copy/paste from the previous PR tbh

Comment (bmwiedemann) [inline scripts/qa_crowbarsetup.sh]: OK, then let's keep it for now and then clean up later

**Pull request merged**